Test/test2/Rlimitouts0.bpl.expect

```diff
@@ -0,0 +1,21 @@
+Prover error: line 105 column 10: max. resource limit exceeded
+Rlimitouts0.bpl(9,16): Verification inconclusive (TestTimeouts0)
```

Comment:
I started the file from a copy of test2/Timeouts0.bpl. It looks reasonably formatted in my visual studio window (and there doesn't seem to have any TAB). I am sure what other things to check.